README.md

```diff
@@ -1,11 +1,33 @@
 This directory contains utilities that facilitate running WDLs through the Broad's Cromwell execution engine on the Google Cloud Platform. 
+# WDL Runner
```

Comment:
Small nit – I find it useful to include the acronym when a term is first used so that a user can refer back to it if they're confused.  So, I'd suggest

> This directory contains utilities that facilitate running WDLs through the Broad's Cromwell execution engine on the Google Cloud Platform (GCP).